src/Simplex/Chat.hs

```diff
@@ -578,8 +578,11 @@ processChatCommand = \case
     withCurrentCall contactId $ \userId ct call ->
       updateCallItemStatus userId ct call receivedStatus Nothing $> Just call
   APIUpdateProfile profile -> withUser (`updateProfile` profile)
-  APISetContactAlias contactId userAlias -> pure CRCmdOk
-  APIUnsetContactAlias contactId -> pure CRCmdOk
+  APISetContactAlias contactId userAlias -> withUser $ \User {userId} -> do
+    ct' <- withStore $ \db -> do
+      ct <- getContact db userId contactId
+      liftIO $ updateContactProfileUserAlias db userId ct userAlias
```

Comment:
This is not user alias - it’s contact alias

Reply:
it's alias given by the user - to distinguish other fields in profile that are defined by contact